target/linux/x86/base-files/lib/preinit/79_move_config

```diff
@@ -2,13 +2,15 @@
 # Copyright (C) 2012-2015 OpenWrt.org
 
 move_config() {
-	local partdev
+	local partdev magic parttype=ext4
 
 	. /lib/upgrade/common.sh
 
 	if export_bootdevice && export_partdevice partdev 1; then
 		mkdir -p /boot
-		mount -t ext4 -o rw,noatime "/dev/$partdev" /boot
+		magic="$(get_magic_vfat "/dev/$partdev")"
+		[ "$magic" = "FAT" ] && parttype=vfat
```

Comment:
I've refactored this further into `part_magic_{fat,efi}` in https://git.openwrt.org/?p=openwrt/staging/ynezz.git;a=commit;h=82610eeff67044fb4a159e246016afb1627cba32, is that ok with you?

Reply:
OK.Should I update this pr?

Reply:
> Should I update this pr?

Not needed, I'm just asking if this modification is fine with you, that's all about it.